### ISSUE TITLE
#Oakland channel gets flooded during project reviews

### ISSUE THREAD

**Issue by EthanJStark** [LearnersGuild/los#114]
### Expected vs. Actual

While projects are being reviewed, I would expect to still be able to view the chat on the Oakland channel. Currently, it's flooded with so many notifications, from every project that gets reviewed, that it's hard to follow important updates.

### Steps to Reproduce

Look at the #Oakland channel during a voting cycle

Ideas: either contain this feed of completed reviews in its own channel, or omit those notifications altogether

**Comment by jeffreywescott** [comment]
Duplicate of #80.
